dandi/download.py

```diff
@@ -578,26 +583,16 @@ def _download_file(
                 and "sha256" in digests
             ):
                 if key_parts[-1].partition(".")[0] == digests["sha256"]:
+                    lgr.debug("already exists - matching digest in filename")
```

Comment:
it is always "not" fun to guess what message is talking about, so better use
```suggestion
                    lgr.debug("%s already exists - matching digest in filename", path)
```

or alike here and elsewhere

Reply:
Got it. Also noticed that there was a `checksum` yield message that would apply when we skip because it matches and added that.